bullet_train-api/app/controllers/api/v1/application_controller.rb

```diff
@@ -0,0 +1,3 @@
+class Api::V1::ApplicationController < ActionController::API
+  include Api::Controllers::Base
+end
```

Comment:
This and the next couple of files are just providing a default implementation of files that we ship in the starter repo. We can think of the files that we ship in the starter repo as being "ejected" versions of these files. I added these in the gem itself so that we don't have to supply these files in `test/dummy` of every singe gem that ends up needing them.

Reply:
I remember this being really annoying because the tests complained about the files not being there. This seems like a good option!

I think it would be helpful to provide a comment in these files briefly explaining the situation so in the future someone on the Bullet Train team doesn't come across it and mistake it for a "duplicated" file in the starter repository.